packages/cli/cli/versions.yml

```diff
@@ -1,3 +1,13 @@
+- changelogEntry:
+    - summary: |
+        Fix issue where the CLI would not parse asyncapi specs within the `api.specs` array in generators.yml.
+      type: fix
+    - summary: |
+        Migrate old spec configuration in generators.yml to the `api.specs[]` format.
+      type: feat
+  irVersion: 55
+  version: 0.53.15
```

Comment:
lets make sure to release this as 0.54.0 since there is technically a break here